benefit/data/sheets/CaseTypeTab/CaseTypeTab-nonprod.json

```diff
@@ -7,6 +7,5 @@
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "caseManagementCategory", "TabFieldDisplayOrder": 5},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "caseAccessCategory", "TabFieldDisplayOrder": 6},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "SearchCriteria", "TabFieldDisplayOrder": 7},
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "caseInfo_sscs", "TabLabel": "Appeal Details", "TabDisplayOrder": 4, "CaseFieldID": "caseManagementLocation", "TabFieldDisplayOrder": 40},
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "listing_requirements_superuser", "TabLabel": "Listing Requirements", "TabDisplayOrder": 24, "CaseFieldID": "dwpIsOfficerAttending", "TabFieldDisplayOrder": 9}
```

Comment:
This has been moved to the appropriate S&L non-prod file:
https://github.com/hmcts/sscs-ccd-definitions/pull/977/files#diff-c15c8c12879d9ba96e7161cf0ce922344e494b31211670f79fbdf15512a4b3e2R6